mmv1/templates/terraform/examples/cloud_run_service_secret_environment_variables.tf.erb

```diff
@@ -0,0 +1,75 @@
+data "google_project" "project" {
+  provider = google-beta
+}
+
+resource "google_secret_manager_secret" "secret" {
+  provider = google-beta
+
+  secret_id = "<%= ctx[:vars]['secret_id'] %>"
+  replication {
+    automatic = true
+  }
+
+  depends_on = [google_project_service.secret-manager]
```

Comment:
Will need to remove this now that the project_service is gone

Reply:
Done everywhere.